etc/profile-m-z/torbrowser-launcher.profile

```diff
@@ -59,7 +59,7 @@ seccomp.block-secondary
 #tracelog # may cause issues, see #1930
 
 disable-mnt
-private-bin bash,cat,cp,cut,dirname,env,expr,file,gpg,grep,gxmessage,id,kdialog,ln,mkdir,mv,python*,rm,sed,sh,tail,tar,tclsh,test,tor-browser,tor-browser-en,torbrowser-launcher,update-desktop-database,xmessage,xz,zenity
+private-bin bash,cat,cp,cut,dirname,env,execdesktop,expr,file,gpg,grep,gxmessage,id,kdialog,ln,mkdir,mv,python*,rm,sed,sh,tail,tar,tclsh,test,tor-browser,tor-browser-en,torbrowser-launcher,update-desktop-database,xmessage,xz,zenity
```

Comment:
`execdesktop` is a script local to the tor-browser distribution (`./Browser/execdesktop`). Is it really supposed to be in the `private-bin` list? (From the manpage it seems that `private-bin` checks distribution's bin paths.) Or are you assuming a case where the portable tor-browser package is repackaged by the linux distribution?)